examples/tcp_listener_fixed_buffers.rs

```diff
@@ -50,7 +50,11 @@ async fn accept_loop(listen_addr: SocketAddr) {
 // A loop that echoes input to output. Use one fixed buffer for receiving and sending the response
 // back. Once the connection is closed, the function returns and the fixed buffer is dropped,
 // getting the fixed buffer index returned to the available pool kept by the registry.
-async fn echo_handler(stream: TcpStream, peer: SocketAddr, registry: FixedBufRegistry) {
+async fn echo_handler<T: IoBufMut>(
```

Comment:
Just a side note. I added this example long before there was a pool layer built on top of fixed buffers and well before I understood how integration tests could be added to crate. It's interesting, for this PR, how just the signature of the handler function had to be modified. I guess that is a pretty standard occurrence in Rust creates.